lib/spotlight/dor/indexer.rb

```diff
@@ -182,6 +182,50 @@ def add_folder_name(sdb, solr_doc)
       end
     end
 
+    concerning :FullTextIndexing do
+      included do
+        before_index :add_object_full_text
+      end
+
+      # search for configured full text files, and if found, add them to the full text (whole document) solr field
+      def add_object_full_text(sdb, solr_doc)
+        object_level_full_text_urls(sdb).each do |file_url|
+          # append content from each file to the field, creating field if it doesn't exist yet (.to_s on nil gives you "")
```

Comment:
as written, if no full text files are found, the solr field will be nil (the case for any project except feigbenbaum for now) There is a test for this:
 https://github.com/sul-dlss/spotlight-dor-resources/blob/issue2-full-text/spec/lib/spotlight/dor/indexer_spec.rb#L569-L574